lib/article_json/import/google_doc/html/text_box_parser.rb

```diff
@@ -22,10 +22,21 @@ def content
             @nodes.map { |node| parse_sub_node(node) }.compact
           end
 
+          # Extract any potential tags, specified in brackets after the Textbox definition
+          # @return [Array[Symbol]]
+          def tags
+            match = /(.*?)[\s\u00A0]+\[(?<tags>.*)\]/.match(@float_node.inner_text)
```

Comment:
Would this only match words in brackets in the first row (expected behavior) or would it match any brackets in a textbox DOM? We'd only expect the tags to be picked up in example 1, but not in 2, right?

```
Textbox: [tag-1 tag-2]
Some textbox content...
```
vs
```
Textbox:
Some textbox content... [tag-1 tag-2]
```

Reply:
Yes, you are right, it would work like in example 1.